### PR TITLE
On Send Confirmation Screen, clicking max spend will always make the top flip input crypto

### PR DESCRIPTION
**Pull request by amanalili**
Task Send Confirmation - flip crypto/fiat amount when user selects max send (https://app.asana.com/0/361770107085503/1134961556780288)

#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [x] Tested on iOS Tablet
- [x] Tested on small Android

### CODE REVIEW

File: src/modules/UI/components/FlipInput/FlipInput2.ui.js [hunk 1]
```diff
@@ -204,6 +205,15 @@ export class FlipInput extends Component<Props, State> {
     }
   }
 
+  componentDidUpdate (prevProps: Props) {
+    if (this.props.toggleFiatOnTop != null) {
+      console.log(this.props.toggleFiatOnTop)
```
Comment (swansontec) [inline src/modules/UI/components/FlipInput/FlipInput2.ui.js]: Please remove `console.log`

File: src/reducers/scenes/SendConfirmationReducer.js [hunk 2]
```diff
@@ -245,6 +247,17 @@ export const transaction = (state: EdgeTransaction | null = null, action: Action
...
 }
 
+export const toggleFiatOnTop = (state: boolean | null = null, action: Action) => {
+  switch (action.type) {
+    case 'UI/SEND_CONFIMATION/TOGGLE_FIAT_ON_TOP': {
+      return action.data
+    }
+
+    default:
+      return state
```
Comment (swansontec) [inline src/reducers/scenes/SendConfirmationReducer.js]: Once this gets set, it never gets unset, which means the flip input would be stuck flipped forever, I think.

File: src/modules/UI/components/FlipInput/FlipInput2.ui.js (hunk 1 above)
Comment (amanalili) [reply src/modules/UI/components/FlipInput/FlipInput2.ui.js]: Fixed

File: src/reducers/scenes/SendConfirmationReducer.js (hunk 2 above)
Comment (amanalili) [reply src/reducers/scenes/SendConfirmationReducer.js]: Yeah I guess so, fixed to default state null

File: src/actions/SendConfirmationActions.js [hunk 3]
```diff
@@ -143,6 +148,7 @@ export const updateMaxSpend = () => (dispatch: Dispatch, getState: GetState) =>
   const walletId = getSelectedWalletId(state)
   const edgeWallet = getWallet(state, walletId)
   const spendInfo = getSpendInfo(state)
+  const toggleCryptoOnTopState = state.ui.scenes.sendConfirmation.toggleCryptoOnTop
```
Comment (paullinator) [inline src/actions/SendConfirmationActions.js]: Do the incrementing in the reducer versus here. The reducer already has access to the current counter and can simply increment it.

File: src/reducers/scenes/SendConfirmationReducer.js [hunk 4]
```diff
@@ -245,6 +247,17 @@ export const transaction = (state: EdgeTransaction | null = null, action: Action
   }
 }
 
+export const toggleCryptoOnTop = (state: number = 0, action: Action) => {
+  switch (action.type) {
+    case 'UI/SEND_CONFIMATION/TOGGLE_CRYPTO_ON_TOP': {
```
Comment (paullinator) [inline src/reducers/scenes/SendConfirmationReducer.js]: This is where you would want to increment the counter instead of in the action

**Comment by paullinator** [comment]
This looks pretty good but I just left a couple small comments which should be easy changes

**Pull request merged**